legend-engine-server/pom.xml

```diff
@@ -282,7 +282,7 @@
         </dependency>
         <dependency>
             <groupId>org.finos.legend.engine</groupId>
-            <artifactId>legend-engine-xt-nonrelationalStore-mongodb-grammar</artifactId>
```

Comment:
either a clear solution is done on local binder (one that is not adding stuff to an uber pure repo), or we could just not add this here.  This change can then be added once there is a cleaner solution on that space.

This will allow the server test cases to run without getting conflicted from mongo local binder.

In other words - lets divide and conquer on this PR, so we can move it fwd on the places we can move it fwd.

Reply:
The grammar is required for studio integration, but now we moved the binder annotated function out to executionPlan-Test module - so there is no break to other modules.